Sources/BrowserServicesKit/Autofill/AutofillUserScript.swift

```diff
@@ -191,11 +197,11 @@ extension AutofillUserScript {
 
             let script = """
             (() => {
-                window.\(methodName)({
+                window.\(methodName) && window.\(methodName)({
```

Comment:
Ensure we only call the secret method if it's actually available.  This is more of a fall back position just in case, to avoid creating errors in the console.